### PR TITLE
Fixes #15428: Wrong exclusion of folder in update.cf for common ncf

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/15428

### CODE REVIEW

File: techniques/system/common/1.0/update.st [hunk 1]
```diff
@@ -273,7 +273,7 @@ bundle agent update_action
     !root_server.(new_promises_available|rudder_ncf_hash_update_repaired)::
       "${g.rudder_ncf}/common"
         copy_from         => remote_unsecured_without_perms("${server_info.policy_server}", "${g.rudder_ncf_origin_common}"),
-        depth_search      => recurse_ignore("inf", @{g.ncf_common_excludedreps}),
+        depth_search      => recurse_ignore("inf", @{g.ncf_local_excludedreps}),
```
Comment (amousset) [inline techniques/system/common/1.0/update.st]: shouldn't it be `excludedreps`?

Comment (ncharles) [reply techniques/system/common/1.0/update.st]: i'm pretty sure we should not copy 50_techniques

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**